catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/BasicTypes.java

```diff
@@ -99,6 +99,8 @@
      */
     public static final AttributeType<Short> SHORT_TYPE;
 
+    public static final AttributeType<HashMap> HASHMAP_TYPE;
```

Comment:
I agree that this should just be `OBJECT_TYPE`, but for what it's worth, I think `HashMap` was used here because `HashMap` implements serializable while `Map` doesn't.